src/api/payment-requests/payment-requests.md

```diff
@@ -171,6 +173,14 @@ corresponds to the GPC brick attributes.
       value: { amount: '8991', currency: 'NZD' },
     })
   }
+  example {
+    title 'Create a Payment Request with redirect url'
+    body ({
+      configId: '5efbe2fb96c08357bb2b9242',
+      value: { amount: '8991', currency: 'NZD' },
+      redirectUrl: 'https://app.centrapay.com',
```

Comment:
can you use a realistic shopping url like https://example.com/store/checkout?cartId=1234